Dockerfile

```diff
@@ -13,22 +13,16 @@
 # limitations under the License.
 
 # Builder image
```

Comment:
It's better add comment in the Dockerfile and README.md to notice it just for development.
And the product image Dockerfile in managed in:
- just the final image: https://github.com/PingCAP-QE/artifacts/blob/main/dockerfiles/products/tidb.Dockerfile
- multi stages dockerfile: https://github.com/PingCAP-QE/artifacts/blob/main/dockerfiles/cd/builders/tidb/Dockerfile

Reply:
Some information has been added.